bin/steps/sqlite3

```diff
@@ -7,9 +7,9 @@ sqlite3_version() {
   if [ "$STACK" = "cedar-14" ]; then
     SQLITE3_VERSION="3.8.2-1ubuntu2.2"
   elif [ "$STACK" = "heroku-16" ]; then
-    SQLITE3_VERSION="3.11.0-1ubuntu1.2"
+    SQLITE3_VERSION="3.11.0-1ubuntu1.3"
```

Comment:
I think we'll maybe need to test 3.2 as well, since that version of DDS used heroku-16, right?

Reply:
Yes, but il will be problematic to release a fix for 3.2 considering the way replicated handle the versions. If a 3.2 client hits the bug and refuses to upgrade we will have to either release something on a different channel or ask the client to use a .buildpacks file pointing to the fixed buildpack. I will discuss this on the issue.